server/test/chokidar-test.ts

```diff
@@ -57,11 +57,11 @@ describe('chokidar', function() {
 
       expect(events).to.have.lengthOf(6);
       expect(events).to.deep.include({ event: 'addDir', path: `${workDir}` });
-      expect(events).to.deep.include({ event: 'addDir', path: `${workDir}/a` });
-      expect(events).to.deep.include({ event: 'addDir', path: `${workDir}/b` });
-      expect(events).to.deep.include({ event: 'addDir', path: `${workDir}/b/c` });
-      expect(events).to.deep.include({ event: 'add', path: `${workDir}/a/ember-cli-build.js` });
-      expect(events).to.deep.include({ event: 'add', path: `${workDir}/b/c/ember-cli-build.js` });
+      expect(events).to.deep.include({ event: 'addDir', path: path.join(workDir, '/a') });
+      expect(events).to.deep.include({ event: 'addDir', path: path.join(workDir, '/b') });
+      expect(events).to.deep.include({ event: 'addDir', path: path.join(workDir, '/b/c') });
```

Comment:
path.join also normalises the path which should change the directory separator. The test that failed because of this is also passing now.